File: 110algoritmos/exercicios010/ex001-Csharp/ex010.cs
using System;

class Program {
    static void Main() {
        double real, dollar;

        Console.Write("Digite quantos reais tem na carteira R$: ");
        real = int.Parse(Console.ReadLine());

        dollar = real/4.64;

        Console.WriteLine("Com R$ " + real + " você pode comprar $ " + dollar.ToString("F") + "dollars");

    }
}

File: 110algoritmos/exercicios010/ex001-Rust/ex010.rs
use std::io;

fn main() {
    let mut real:String = String::from("");

    println!("Digite quantos reais tem na carteira: R$ ");
    io::stdin()
        .read_line(&mut real)
        .expect("Erro ao ler real");
    let real:f32 = real.trim().parse().expect("Erro ao converter real");

    let dollar = real/4.61;

    println!("Com R$ {} reais você pode comprar $ {} dollares", real, dollar);

}